src/cpp/session/resources/schema/user-state-schema.json

```diff
@@ -4,6 +4,139 @@
     "title": "RStudio User State",
     "type": "object",
     "properties": {
+        "general": {
+            "type": "object",
+            "properties": {
+                "ignoredUpdateVersions": {
+                    "type": "array",
+                    "description": "A list of ignored updates. Ignored updates will not trigger an available update.",
+                    "items": {
+                        "type": "string"
+                    }
+                },
+                "clipboardMonitoring": {
```

Comment:
FWIW this preference might not be necessary anymore with Electron.

Reply:
Good point, I'll take another look at how it's used and see about cleaning this up.